src/main/scala/com/fulcrumgenomics/vcf/filtration/FilterSomaticVcf.scala

```diff
@@ -52,17 +52,40 @@ import scala.collection.immutable.ListMap
     |
     |## Available Filters
     |
+    |### A-tailing Artifact Filter
+    |
+    |The A-tailing artifact filter attempts to measure the probability that a SNV is the product of
```

Comment:
I might spell it out to be extra clear we are talking about "mismatches" and not true variants.

```suggestion
    |The A-tailing artifact filter attempts to measure the probability that a single-nucleotide mismatch is the product of
```